make/env.mk

```diff
@@ -40,7 +40,7 @@ endif
 TAG := # make sure tag is never injectable as an env var
 RELEASE_GOTAGS := release
 
```

Comment:
(not about this line) Now that the code looks good, there are CI failures to take care of.

The Konflux one looks like there's an outage https://github.com/stackrox/stackrox/pull/12909/checks?check_run_id=33221201821

The GHA ones are, sigh, interesting.
Take this one for example https://github.com/stackrox/stackrox/actions/runs/11920074741/job/33221202124?pr=12909#step:6:15
There's `GOEXPERIMENT=cgocheck2` above which gets handled well and there's
```
go: unknown GOEXPERIMENT strictfipsruntime
```

Apparently, `GOEXPERIMENT` is a special value recognized by Go toolchain - <https://pkg.go.dev/internal/goexperiment>. A vanilla compiler used in GHA builds does not know `strictfipsruntime` and therefore is failing.

Only Red Hat-patched toolchain in `openshift-golang-builder` would be quietly happy about `strictfipsruntime`, I suppose.
Unfortunately, we need to go back to the revision where you provided `GOEXPERIMENT` in Konflux images. I think, it would make sense to keep `GOTAGS` `strictfipsruntime` also next to that so that it's easier to get rid of the redundant one (when it's time).

Sorry that I pushed you all the way here and now we need to go back.